operate/ledger/profiles.py

```diff
@@ -21,62 +21,41 @@
 
 import typing as t
 
+from autonomy.chain.constants import CHAIN_PROFILES, DEFAULT_MULTISEND
+
 from operate.constants import NO_STAKING_PROGRAM_ID, ZERO_ADDRESS
 from operate.operate_types import Chain, ContractAddresses
 
 
-CONTRACTS: t.Dict[Chain, ContractAddresses] = {
-    Chain.GNOSIS: ContractAddresses(
-        {
-            "service_manager": "0x04b0007b2aFb398015B76e5f22993a1fddF83644",
-            "service_registry": "0x9338b5153AE39BB89f50468E608eD9d764B755fD",
-            "service_registry_token_utility": "0xa45E64d13A30a51b91ae0eb182e88a40e9b18eD8",
-            "gnosis_safe_proxy_factory": "0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE",
-            "gnosis_safe_same_address_multisig": "0x6e7f594f680f7aBad18b7a63de50F0FeE47dfD06",
-            "multisend": "0x40A2aCCbd92BCA938b02010E17A5b8929b49130D",
-        }
-    ),
-    Chain.OPTIMISM: ContractAddresses(
+# TODO: Refactor, remove the usage of CONTRACTS and use CHAIN_PROFILES from Open Autonomy instead.
+CHAINS = [
```

Comment:
I believe there's no `POLYGON` here, but it is there in the code below. In general I would add all the EVM chains we are operating one - here it's the addition of `POLYGON` and `ARBITRUM`